src/app/components/client/ProgressCard.tsx

```diff
@@ -15,67 +15,25 @@ import { Button } from "../server/Button";
 import { useL10n } from "../../hooks/l10n";
 import ExploringLaptopPlus from "./assets/exploring-laptop-check.svg";
 import ExploringLaptopMinus from "./assets/exploring-laptop-minus.svg";
-import SparklingCheck from "./assets/sparkling-check.svg";
```

Comment:
Is this svg orphaned now?

```sh
git grep "sparkling-check" | wc -l # 0
```

Reply:
In other words: probably a good idea to delete the file. Even if we were to use it in the future, we'd probably download it from the Figma again, rather than realising it might already be present in the repository.